### PR TITLE
pythonPackages.pypandoc: 1.4 -> 2019-03-01

### PR DESCRIPTION
**Pull request by Twey**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/pypandoc/default.nix [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
-  version = "1.4";
+  version = "2019-03-01";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "e914e6d5f84a76764887e4d909b09d63308725f0cbb5293872c2c92f07c11a5b";
+  src = fetchFromGitHub {
+    owner = "bebraw";
+    repo = "pypandoc";
+    rev = "87912f0f17e0a71c1160008df708c876d32e5819";
```
Comment (FRidh) [inline pkgs/development/python-modules/pypandoc/default.nix]: Why a seemingly arbitrary commit?

**Comment by Twey** [comment]
It's just the head of master at the time, and works with the version of Pandoc we have in nixpkgs. We can bump it to a release next time it releases, but until then there's not much point having the released version around when it doesn't work.

**Comment by FRidh** [comment]
It's worth mentioning the choice of some revision in the code.

**Comment by dotlambda** [comment]
87912f is from 2018-06-18. You might also want to prefix the version with `unstable-`.

**Comment by aanderse** [comment]
@Twey It looks like you're really close to a merge. If you can address the comments above, specifically about adding a comment on choice of revision and the version string I'm sure everyone would be happy to commit.

PS. from the nixpkgs manual, regarding version string:

> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".

**Comment by sternenseemann** [comment]
Would be great to get this merged soon as I am currently trying to a package that depends on `pypandoc`.

**Comment by bennofs** [comment]
pypandoc update was merged with #75840

**Pull request closed without merging**